pkg/volume/plugins.go

```diff
@@ -42,6 +42,13 @@ type VolumeOptions struct {
 	RootContext string
 }
 
+// System is an interface that abstracts certain operations on a host operating
+// system that a volume might want to execute.
+type System interface {
```

Comment:
Host is your gateway to the rest of the app (think of it as if the plugin was a .so file).  The host application provides an implmentation of Mounter and a function to write files in the filesystem